corehq/apps/hqwebapp/management/commands/synch_phone_nums.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.users.models import CommCareUser
+from corehq.apps.sms.tasks import sync_user_phone_numbers as sms_sync_user_phone_numbers
+from corehq.form_processor.models import CommCareCaseSQL
+from corehq.messaging.tasks import sync_case_for_messaging
+from corehq.util.log import with_progress_bar
+from corehq.sql_db.util import get_db_aliases_for_partitioned_query
+
+
+class Command(BaseCommand):
+    help = "Resync all contacts' phone numbers for given projects"
+
+    def add_arguments(self, parser):
+        parser.add_argument('-p', '--projects', type=str, help='Comma separated project list')
+
+    def handle(self,  **kwargs):
+        projects = kwargs['projects'].split(',')
+        db_aliases = get_db_aliases_for_partitioned_query()
+        db_aliases.sort()
+
+        for domain in projects:
+            print("Resync all contacts' phone numbers for project %s  " % domain)
```

Comment:
requires `from __future__ import print_statement` at top of file

Reply:
i think you meant `from __future__ import print_function`. 
Although adding that also did not remove the lynting errors

Reply:
yes, you're right. Some lint errors are not automatically removed (if they're not near a line that was changed since they were added. You can manually resolve them though.